docs/application/dotnet/guides/overview.md

```diff
@@ -28,11 +28,11 @@ You can use the following TizenFX features in your .NET applications:
 
   The application management feature introduces how to handle applications and packages. You can see application resources and communication methods between applications.
```

Comment:
As @mijin85cho pointed out this file is being replaced with a new index.md file in PR 1483 you would have include that file and make changes accordingly in that file.

Reply:
Okay, will wait for your fix to approve this PR. I will try to get 1483 merged by today or tomorrow.

Reply:
@mihashco this file has to be removed